src/utils/utils.js

```diff
@@ -617,3 +617,58 @@ export function injectNotifications(tab_id, importExport = false) {
 export function isCliqzOffer(offer) {
 	return (offer && offer.origin === 'cliqz' && offer.type === 'offers' && offer.data);
 }
+
```

Comment:
@benstrumeyer Please move all of these functions back to app/panel/utils. Background utils should only be imported by modules in the background (src) context. This prevents adding unnecessary piggyback imports to the React app.  If you need to communicate between app/ and src/ code, you can either use message passing or add light functions to the import-free src/utils/common.js file.  In this case, you only need this functions in the React app so they should live under app directory.

Reply:
That makes sense. I moved the functions back to where they were originally

Reply:
Still need to remove these functions.

Reply:
I didn't save the file and all of these errors slipped through. Removed these guys, sorry about that.